crates/solver-types/src/utils/builders/order.rs

```
@@ -0,0 +1,225 @@
+//! Order builder utilities for creating test and production Order instances.
+
+use crate::{Address, FillProof, Order, OrderStatus, TransactionHash};
+
+/// Builder for creating Order instances with sensible defaults.
+///
+/// This builder provides a fluent interface for constructing Order objects,
+/// particularly useful for testing and creating orders with common patterns.
+#[derive(Debug, Clone)]
+pub struct OrderBuilder {
+	id: String,
+	standard: String,
+	status: OrderStatus,
+	solver_address: Address,
+	created_at: u64,
+	updated_at: u64,
+	data: serde_json::Value,
+	quote_id: Option<String>,
+	input_chain_ids: Vec<u64>,
+	output_chain_ids: Vec<u64>,
+	execution_params: Option<crate::ExecutionParams>,
+	prepare_tx_hash: Option<TransactionHash>,
+	fill_tx_hash: Option<TransactionHash>,
+	post_fill_tx_hash: Option<TransactionHash>,
+	pre_claim_tx_hash: Option<TransactionHash>,
+	claim_tx_hash: Option<TransactionHash>,
+	fill_proof: Option<FillProof>,
+}
+
+impl Default for OrderBuilder {
+	fn default() -> Self {
```

If we're using this for production (not just testing), I wouldn't default to random values.. If we are using it _just_ for testing, maybe we need to switch them to a `tests/` folder to imply that?

---

This goes for all builders btw

---

We are using only for testing atm, good suggestion, I can move all builder inside /test folder